packages/integration-objects/test/ValidateFieldsForRateTypeDesc.test.js

```diff
@@ -0,0 +1,97 @@
+/* Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. */
+const { ValidationError } = require('../srv/exceptions/validation-error');
+const { validateFieldsForExchangeRateTypeDescription } = require('../srv/validations/ValidateFieldsForRateTypeDesc');
+const ErrorStatuses = require('../srv/utils/ErrorStatuses');
+const RateTypeExtensionConstants = require('../srv/utils/RateTypeExtensionConstants');
+
+describe('validate fields of rate type description', function () {
+  it('does not return any error', function () {
+    const reqObj = {
+      locale: 'en',
+      exchangeRateTypeDescription: 'sample'
+    };
+    const params = {
+      ID: 'id'
+    };
+    expect(() => validateFieldsForExchangeRateTypeDescription(reqObj, params));
```

Comment:
This does not test anything. If you want to check that this does not throw you would have to add something like `.not.toThrow()`

Reply:
Updated